wpilibc/src/main/native/include/frc/livewindow/LiveWindow.h

```diff
@@ -68,7 +68,7 @@ class LiveWindow {
    *
    * @param sendable component to remove
    */
```

Comment:
Please add a documentation comment here for the meaning of the return type (eg what is it and what does a true value mean?).